gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/notifiers/impl/WebNotifierServiceImpl.java

```diff
@@ -215,6 +219,7 @@ public void handle(Throwable throwable) {
             future.get();
         } catch (InterruptedException | ExecutionException e) {
             LOGGER.error(e.getMessage(), e);
+            Thread.currentThread().interrupt();
```

Comment:
I don't think we should interrupt the current thread in case of an `ExecutionException`